manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.installer.yaml

```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: LibreWolf.LibreWolf
```

Comment:
Please update your automation to add the Schema Headers and also to use the 1.1.0 schema
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json

PackageIdentifier: LibreWolf.LibreWolf
```